db/aliasManager.go

```diff
@@ -0,0 +1,242 @@
+package db
+
+import (
+	"context"
+	"strings"
+	"strconv"
+	"errors"
+	"fmt"
+	"math/rand"
+	tinycrypt "github.com/uclaacm/teach-la-go-backend-tinycrypt" 
+	"cloud.google.com/go/firestore"
+	"google.golang.org/api/iterator"
+)
+
+type Shard struct {
+	Count int64
+}
+
+var crypt = tinycrypt.Encrypter{
+	[]uint64 {
+		0x33684192D,
+		0x28DAB6A5A,
+		0xA928A9246,
+		0x224D23A42,
+	},
+}
+
+
+
+
+func (d *DB) InitShards(ctx context.Context, path string)(error){
+
+	// get the document that has the counters
+	doc := d.Collection(path).Doc("--shards--")
```

Comment:
Might be worthwhile making the shard document name a constant.

Reply:
new constant `shard_name` created